ローカルでユニットテストを実行するとタイムゾーンのずれで失敗する

![image](https://github.com/user-attachments/assets/06c66104-2e8b-43bc-8797-3da51ebef68b)

```plaintext

⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯ Failed Tests 2 ⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯

 FAIL  tests/unit/lib/basic/date.spec.ts > getISOFullDayString > can get iso full day (2)
AssertionError: expected '2001-04-03' to be '2001-04-04' // Object.is equality

Expected: "2001-04-04"
Received: "2001-04-03"

 ❯ tests/unit/lib/basic/date.spec.ts:57:47
     55|   })
     56|   it('can get iso full day (2)', () => {
     57|     expect(getISOFullDayString(defaultDate2)).toBe('2001-04-04')
       |                                               ^
     58|   })
     59| })

⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯[1/2]⎯

 FAIL  tests/unit/lib/searchMessage/queryParser.spec.ts > parseQuery > can parse query with date-filter
AssertionError: expected '2021-01-22T15:00:00.000Z' to be '2021-01-23T00:00:00.000Z' // Object.is equality

Expected: "2021-01-23T00:00:00.000Z"
Received: "2021-01-22T15:00:00.000Z"

 ❯ tests/unit/lib/searchMessage/queryParser.spec.ts:54:31
     52|     expect(normalizedQuery).toBe(query)
     53|     expect(queryObject.word).toBe('lorem ipsum')
     54|     expect(queryObject.after).toBe('2021-01-23T00:00:00.000Z')
       |                               ^
     55|   })
     56|   it('can parse query with in-filter (prefix: `in:`)', async () => {
```

--- a/vite.config.ts
+++ b/vite.config.ts
@@ -129,6 +129,9 @@ export default defineConfig(({ command, mode }) => ({
     brotli()
   ],
   test: {
+    env: {
+      TZ: 'UTC'
+    },
     include: ['tests/unit/**/*.spec.ts'],
     globals: true,
     setupFiles: ['tests/unit/setup.ts', 'tests/unit/expectExtends.ts'],

